notebook/nbextensions.py

```diff
@@ -299,118 +412,356 @@ def _config_file_name_default(self):
     def install_extensions(self):
         if len(self.extra_args)>1:
             raise ValueError("only one nbextension allowed at a time.  Call multiple times to install multiple extensions.")
-        install_nbextension(self.extra_args[0],
+        install = install_nbextension_python if self.python else install_nbextension
+        install(self.extra_args[0],
             overwrite=self.overwrite,
             symlink=self.symlink,
             verbose=self.verbose,
             user=self.user,
+            sys_prefix=self.sys_prefix,
             prefix=self.prefix,
-            destination=self.destination,
-            nbextensions_dir=self.nbextensions_dir,
+            nbextensions_dir=self.nbextensions_dir
         )
     
     def start(self):
         if not self.extra_args:
-            for nbext in jupyter_path('nbextensions'):
-                if os.path.exists(nbext):
-                    print("Notebook extensions in %s:" % nbext)
-                    for ext in os.listdir(nbext):
-                        print(u"    %s" % ext)
+            sys.exit('Please specify an nbextension to install')
         else:
             try:
                 self.install_extensions()
             except ArgumentConflict as e:
-                print(str(e), file=sys.stderr)
-                self.exit(1)
+                self.log.info(str(e), file=sys.stderr)
+                sys.exit(1)
 
+_uninstall_flags = {
+    "debug" : ({
+        "UninstallNBExtensionApp" : {
+            "verbose" : 2,
+        }}, "Extra output"
+    ),
+    "quiet" : ({
+        "UninstallNBExtensionApp" : {
+            "verbose" : 0,
+        }}, "Minimal output"
+    ),
+    "user" : ({
+        "UninstallNBExtensionApp" : {
+            "user" : True,
+        }}, "Install to the user's Jupyter directory"
+    ),
+    "sys-prefix" : ({
```

Comment:
Hum, this "feel" to me as a flag that would require a value, so I would expect `--sys-prefix=...` maybe `--use-sys-prefix` ? But I'm not even sure.
Thoughts?